sound/soc/sof/intel/hda-dsp.c

```diff
@@ -617,6 +617,13 @@ static int hda_suspend(struct snd_sof_dev *sdev, bool runtime_suspend)
 #endif
 	int ret, j;
 
+	/*
+	 * The memory used for IMR boot looses it's content in deeper than S3 state
```

Comment:
loses its content

Reply:
Also a typo in commit message "instead of..."